docs-source/examples/three_legged_oauth.rst

```diff
@@ -0,0 +1,191 @@
+Three Legged OAuth with Flask
+-----------------------------
+
+This type of authorization is used for web login with a server-side
+application. For example, a Django app or other application server handles
+requests.
+
+This example uses Flask, but should be easily portable to other application
+frameworks.
+
+
+Components
+~~~~~~~~~~
+
+There are two components to this application: login and logout.
+
+Login sends a user to Globus Auth to get credentials, and then may act on the
+user's behalf.
+Logout invalidates server-side credentials, so that the application may no
+longer take actions for the user, and the client-side session,
+allowing for a fresh login if desired.
+
+Register an App
+~~~~~~~~~~~~~~~
+
+Go to developers.globus.org and register a new application.
```

Comment:
This should probably be a hyper link. Also I was wondering if all the examples should point here? Its much easier to find the registration form through this link, although there isn't an explanation of what Native vs non-Native apps are if we don't send them to the Auth docs.